src/status/libstatus/settings.nim

```diff
@@ -113,3 +115,24 @@ proc getMailservers*():JsonNode =
   let fleet = getSetting[string](Setting.Fleet, $Fleet.PROD)
   result = callPrivateRPC("mailservers_getMailservers").parseJSON()["result"]
 
+proc getWakuVersion*():int =
+  let nodeConfig = getNodeConfig().parseJSON()
+  if nodeConfig["WakuConfig"]["Enabled"].getBool():
+    return 1
+  if nodeConfig["WakuV2Config"]["Enabled"].getBool():
+    return 2
+  return 0
+
+proc setWakuVersion*(newVersion: int) =
+  let nodeConfig = getNodeConfig().parseJSON()
+  if newVersion == 1:
+    nodeConfig["WakuConfig"]["Enabled"] = newJBool(true)
+    nodeConfig["WakuV2Config"]["Enabled"] = newJBool(false)
+    nodeConfig["NoDiscovery"] = newJBool(false)
+    nodeConfig["Rendezvous"] = newJBool(true)
+  else:
+    nodeConfig["WakuConfig"]["Enabled"] = newJBool(false)
+    nodeConfig["WakuV2Config"]["Enabled"] = newJBool(true)
+    nodeConfig["NoDiscovery"] = newJBool(true)
+    nodeConfig["Rendezvous"] = newJBool(false)
```

Comment:
This is probably incorrect but at the moment I'm not sure how `NoDiscovery` and `Rendezvous` should work with WakuV2